README.md

```diff
@@ -155,6 +357,33 @@ successfully deployed cosmsodb database and collections
 
 ```
 
+## Deploy
+
+When you are ready to deploy a new release, you need to synch 
+the source code in the Git repository to the App Service (or to Azure Functions).
+
+Azure gives you the option to configure continuos deployment from a GitHub branch,
+automatically triggering a new deploy through a webhook when changes are pushed.
+To do that you must give your Azure subscription access to your
+GitHub account in order to set up the webhook:  
+https://docs.microsoft.com/en-us/azure/azure-functions/functions-continuous-deployment
+
+We chose to not setup this kind of continous deployment, but
+to provide scripts that, when launched from the command line, 
```

Comment:
`scripts` -> `a script` ?

Reply:
btw shouldn't these instructions be in the respective app repos?

Reply:
the script work with the environment (and json) configurations contained in this repo.
we can link to this repo from the other ones.